pkg/controllers/dynakube/activegate/internal/authtoken/reconciler.go

```diff
@@ -162,3 +165,7 @@ func (r *Reconciler) conditionSetSecretCreated(secret *corev1.Secret) {
 
 	setAuthSecretCreated(r.dk.Conditions(), ActiveGateAuthTokenSecretConditionType, "secret created "+days+" day(s) ago, token:"+tokenPublicPart)
 }
+
+func (r *Reconciler) query() k8ssecret.QueryObject {
```

Comment:
Technically, it's only about `querySecret` and not more generic `query`.

Reply:
true true, but I wanted something short, I would call it `secretQuery`?